testing/iprange/APKBUILD

```diff
@@ -0,0 +1,56 @@
+# Contributor: Jonny Tyers <jtyers@gmail.com>
+# Maintainer: Jonny Tyers <jtyers@gmail.com>
+pkgname=iprange
+pkgver=1.0.3
+pkgrel=0
+pkgdesc="manages IP ranges (part of FireHOL)"
+url="https://github.com/firehol/iprange"
+arch="all"
+license="GPL2"
+depends=""
+depends_dev=""
+makedepends="$depends_dev"
+install=""
+subpackages="$pkgname-doc"
+source="https://firehol.org/download/iprange/releases/v1.0.3/iprange-1.0.3.tar.bz2"
+
+_builddir="$srcdir"/$pkgname-$pkgver
+prepare() {
```

Comment:
abuild in 3.5-stable and edge now provides default_prepare that applies patches. This can be removed.